addons/hr_payroll/models/hr_benefit.py

```diff
@@ -81,11 +81,11 @@ def _check_if_error(self):
         undefined_type.write({'display_warning': True})
         conflict = self._mark_conflicting_benefits(min(self.mapped('date_start')), max(self.mapped('date_stop')))
         conflict_with_leaves = self._compute_conflicts_leaves_to_approve()
-        return undefined_type or conflict or conflict_with_leaves
+        return undefined_type + conflict + conflict_with_leaves
```

Comment:
What about union instead of concatenation?

Reply:
Good idea, it will be corrected in next rebase
Thanks